src/main/java/org/cryptomator/cli/Args.java

```diff
@@ -79,14 +94,33 @@ public int getPort() {
 	}
 
 	public Set<String> getVaultNames() {
-		return vaultPaths.keySet().stream().filter(vaultPasswords::containsKey).map(String.class::cast).collect(Collectors.toSet());
+		Set<String> filteredVaults = vaultPaths.keySet().stream().filter(vaultPasswords::containsKey).map(String.class::cast).collect(Collectors.toSet());
```

Comment:
I think it would be simpler to just use `filter(this::hasPasswordOrPasswordFile)` with
```java
private boolean hasPasswordFile(String vaultPath) {
    return vaultPasswords.containsKey(vaultPath) || vaultPasswordfiles.containsKey(vaultPath);
}
```